php/tests/UndefinedTest.php

```diff
@@ -1,17 +1,21 @@
 <?php
 
-require_once('test_util.php');
```

Comment:
You can leave this file unchanged, since we don't actually test it.

Reply:
But why not fix it too if it's in the repository? The point here was to clean up as much as possible. If it's not required it can be deleted. Or if it has a purpose, it should have the same quality as other similar files.